components/org.wso2.carbon.identity.common/src/main/java/org/wso2/carbon/identity/common/jdbc/JdbcTemplate.java

```diff
@@ -45,27 +45,47 @@ public JdbcTemplate(DataSource dataSource) {
      *
      * @param query     the SQL query with the parameter placeholders.
      * @param rowMapper Row mapper functional interface
+     * @param <E> type of the Exception
```

Comment:
@param <E> type of the Exception which may be thrown by the "RowMapper", and which supposed to be handled by the caller method.